ivadomed/loader/utils.py

```diff
@@ -141,7 +142,7 @@ def get_new_subject_file_split(df, split_method, data_testing, random_seed,
             raise ValueError("All lists in subject_selection parameter should have the same length.")
 
         sampled_dfs = []
-        random.seed(random_seed)
+        set_seed(random_seed)
```

Comment:
Just to recap, this is setting the seed for all dependencies base on the  `"split_dataset": {"random_seed": 6}` in the config JSON right?

Reply:
Right